commands/upgrade/__init__.py

```diff
@@ -113,6 +113,11 @@ def upgrade(args, breadcrumbs):
 
     if workflow.failure:
         sys.exit(1)
+    elif not args.resume:
+        sys.stdout.write(
+            'Reboot the system to continue with the upgrade.'
+            ' This might take a while depending on the system configuration.\n'
```

Comment:
Nit: Could we please not start the next sentence with a whitespace, but rather finish the previous one with it? Other messages cling to that style and imho it's easier to read through.

`Reboot the system to continue with the upgrade. '`
`'This might take a while depending on the system configuration.\n'`